code/game/objects/items.dm

```diff
@@ -276,12 +276,13 @@
 
 // called when this item is removed from a storage item, which is passed on as S. The loc variable is already set to the new destination before this is called.
 /obj/item/proc/on_exit_storage(obj/item/storage/the_storage)
-	SEND_SIGNAL_OLD(the_storage, COMSIG_STORAGE_TAKEN, src, the_storage)
+	SEND_SIGNAL(the_storage, COMSIG_STORAGE_TAKEN, src, the_storage)
```

Comment:
```suggestion
	SEND_SIGNAL(the_storage, COMSIG_STORAGE_TAKEN, src)
```
The first arg of the new `SEND_SIGNAL()` is passed as the first arg in the registered proc. Same thing applies to `on_enter_storage()`.